README.md

```diff
@@ -1,12 +1,16 @@
 # type-errors-pretty
 
 ![pretty-bug](https://user-images.githubusercontent.com/4276606/61183911-eda86300-a679-11e9-9b8e-34c129469075.png)
-[![Build status](https://img.shields.io/travis/chshersh/type-errors-pretty.svg?logo=travis)](https://travis-ci.org/chshersh/type-errors-pretty)
+
+[![GitHub CI](https://github.com/chshersh/type-errors-pretty/workflows/CI/badge.svg)](https://github.com/chshersh/type-errors-pretty/actions)
+[![Build status](https://api.travis-ci.com/chshersh/type-errors-pretty.svg?logo=travis)](https://travis-ci.com/chshersh/type-errors-pretty)
```

Comment:
This will have the Travis logo, they have changed it.

```suggestion
[![Build status](https://img.shields.io/travis/com/chshersh/type-errors-pretty/master?logo=travis)](https://travis-ci.com/chshersh/type-errors-pretty)
```